### ISSUE TITLE
Blockquotes / Headings should be applicable in lists

### ISSUE THREAD

**Issue by bedeoverend** [SimplaElements/simpla-richtext-behavior#18]
Currently blockquotes / headings formatters aren't applicable when highlighting a list item. They should be applicable and the behaviour should be to split the list in two at the given list item, then insert a block with the contents of the highlighted list item between the two.

![example](http://g.recordit.co/TtIefRsf5C.gif)

**Comment by bedeoverend** [comment]
cc/ @seaneking what do you think of this behaviour? Not sure if there's another possible one, but at least need some behaviour - right now it's just blindly broken.

**Comment by madeleineostoja** [comment]
Yep that behavior looks pretty intuitive to me. Though I'd mark this as a low priority item, especially as the logic seems reasonably complex. Very minor use-case, wanting to make a list item a heading or a blockquote, it'd be more for people messing around seeing how things break.

**Comment by madeleineostoja** [comment]
I think an easier, and still perfectly reasonable, solution to this is to make `heading` and `blockquote` `applicable: false` in lists, and then use that to hide the tools in article. That could be done now and looks perfectly stable, IMO.

Unless I'm misunderstanding and the behavior you proposed is super easy to do.

**Comment by bedeoverend** [comment]
Nah, its got some work in it. They're already set to `applicable: false` so we can hide the tools now. I'll update that in article, but yup leave this open for future implementation.

**Comment by madeleineostoja** [comment]
👍 Will downgrade to enhancement, since the new current behavior is perfectly stable.
